flink-libraries/flink-state-processing-api/src/main/java/org/apache/flink/state/table/SavepointConnectorOptions.java

```diff
@@ -35,6 +35,7 @@ public class SavepointConnectorOptions {
     public static final String STATE_TYPE = "state-type";
     public static final String MAP_KEY_FORMAT = "map-key-format";
     public static final String VALUE_FORMAT = "value-format";
+    public static final String VALUE_TYPE_INFO_FACTORY = "value-type-info-factory";
```

Comment:
I think we should also add `key-type-info-factory` to handle pojo/custom Map state keys. While this is not extremely common it's otherwise fully supported and very easy to do from a user perspective

Reply:
Fair point no to limit users.